### PR TITLE
Graphql schema

### LINKED ISSUE #2: Graphql schema
Create a detailed graphql schema with all relevant entities and functions

### PR DESCRIPTION
**Pull request by biosfood**

### CODE REVIEW

File: v1/schema.graphqls [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+"""
+A user's position relative to some fixed point on the map.
+The floor number indicates the level of the building the position is pointing to.
+A floor value of 0 means ground level.
+"""
+type Position {
+  x: Float!
+  y: Float!
+  floor: Int!
```
Comment (biosfood) [inline v1/schema.graphqls:20]: Are you all okay with this or should we do something else (figuring out the floor from GPS coordinates will not be super easy but should be doable). An alternative would be to just use a float for z direction as well.

File: v1/schema.graphqls [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+type LoginResult {
+  token: String
+  status: LoginStatus
+}
+
+type Mutation {
+  login(username: String!, password: String!): LoginResult!
+  """
+  Return value: success
```
Comment (biosfood) [inline v1/schema.graphqls:78]: Is this understandable enough?

File: v1/schema.graphqls (hunk 1 above)
Comment (Lemonatix) [reply v1/schema.graphqls:20]: Seems like the best option here. I assume the GPS will be able to determine the integer designated for a floor. A float does not seem necessary.

File: v1/schema.graphqls (hunk 2 above)
Comment (Lemonatix) [reply v1/schema.graphqls:78]: It is kept very simple, it is very understandable.

**Review by Lemonatix — APPROVED** [review]
Good GraphQL schema for definining position and login of a user

**Pull request merged**